### PR TITLE
Add snapshot testing configuration and add snapshot tests for all existing components

### PR DESCRIPTION
**Pull request by jellebouwman**

### CODE REVIEW

File: components/timesheet-entry/__tests__/TimesheetEntry-test.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+const timesheetEntry = {
+  employer: 'Port of Rotterdam',
+  endTime: '2018-12-13T18:30:00.000Z',
+  id: 1,
+  startTime: '2018-12-13T18:00:00.000Z'
+};
+
+const handleDeleteTimesheetEntry = (timesheetEntryId) => {
+  this.props.onDeleteTimesheetEntry(timesheetEntryId);
```
Comment (boris-arkenaar) [inline components/timesheet-entry/__tests__/TimesheetEntry-test.jsx]: You could leave this function empty.

File: components/timesheet-input/__tests__/TimesheetInput-test.jsx [hunk 2]
```diff
@@ -24,3 +24,15 @@ test('ComponentName', () => {
     }
   });
 });
+const handleAddTimesheetEntry = newEntry => this.props.onPostTimesheetEntry(newEntry);
```
Comment (boris-arkenaar) [inline components/timesheet-input/__tests__/TimesheetInput-test.jsx]: Leave the function empty `() => {}`.

File: components/timesheet/__tests__/Timesheet-test.jsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Timesheet from '../Timesheet';
+
+import {
+  deleteTimesheetEntry,
```
Comment (boris-arkenaar) [inline components/timesheet/__tests__/Timesheet-test.jsx]: No need to use these imports, right?

File: components/timesheet-entry/__tests__/__snapshots__/TimesheetEntry-test.jsx.snap [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      className="timesheet-entry__delete-icon"
+    />
+    Delete
+  </button>
+  <p
+    className="timesheet-entry__time"
+  >
+    7:00 PM
+           -
+           7:30 PM
```
Comment (RikFrieling) [inline components/timesheet-entry/__tests__/__snapshots__/TimesheetEntry-test.jsx.snap]: Maybe you could format this a bit nicer, the indentations on 26/27 is a bit much

Comment (boris-arkenaar) [reply components/timesheet-entry/__tests__/__snapshots__/TimesheetEntry-test.jsx.snap]: This is a snapshot file @RikFrieling and is automatically generated 😉

**Pull request merged**